scripts/library.sh

```diff
@@ -327,21 +327,28 @@ function report_go_test() {
   return ${failed}
 }
 
-# Install Knative Serving in the current cluster.
-# Parameters: $1 - Knative Serving manifest.
-function start_knative_serving() {
```

Comment:
This will break smoke serving tests, please don't remove this function.

Reply:
ok